domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -308,10 +309,21 @@ class StateRetriever @Inject constructor(
       "FractionInput" -> InteractionObject.newBuilder()
         .setFraction(parseFraction(inputJson.getJSONObject(keyName)))
         .build()
+      "DragDropAndSort" -> InteractionObject.newBuilder()
```

Comment:
I can you add tests for this new code? I know that we don’t have tests yet for this file, but this is a good opportunity to introduce those at least for the new functionality.

Reply:
Tests were added.